converter/src/main/java/gov/cms/qpp/conversion/correlation/model/Config.java

```diff
@@ -0,0 +1,55 @@
+package gov.cms.qpp.conversion.correlation.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+	"decodeLabel",
+	"encodeLabels",
+	"goods"
+})
+public class Config {
```

Comment:
Even though it seems more verbose, consider naming this something like `CorrelationConfig`.  `Config` is rather generic and forces us to look at the package structure to figure out what this is configuration for.

Reply:
Done